includes/class-wc-product-tables-install.php

```diff
@@ -49,6 +49,8 @@ public static function activate() {
 			  `type` varchar(100) NULL default 'simple',
 			  `virtual` tinyint(1) NULL default 0,
 			  `downloadable` tinyint(1) NULL default 0,
+			  `download_limit` int(11) default NULL,
```

Comment:
This is exactly what I'm afraid to do, moving to this table, because once here, will be very hard to move this data again for a third time.

Reply:
I agree that moving this data is not trivial but I really don't think we should change to download limit and download expiry in this plugin before we make this change in core. This is just my personal opinion.

Reply:
But this is exactly the reason why we started doing it, to not have it in this table and to avoid the trouble of moving this data twice.